src/dlls/dbgshim/dbgshim.h

```diff
@@ -34,6 +34,14 @@ RegisterForRuntimeStartup(
     __in PVOID parameter,
     __out PVOID *ppUnregisterToken);
 
+EXTERN_C HRESULT
+RegisterForRuntimeStartupEx(
```

Comment:
>RegisterForRuntimeStartupEx [](start = 0, length = 27)

I think you need to add the new functions to src/dlls/dbgshim/dbgshim_unixexports.src